tests/correlated_features.py

```diff
@@ -1,75 +1,84 @@
 import numpy as np
 import pandas as pd
+from xgboost import XGBRegressor
 
-from tests.test_superclass import Test
-from tests.util import is_ok
+from .test_superclass import Test
+from .util import is_ok
```

Comment:
```suggestion
from tests.test_superclass import Test
from tests.util import is_ok
```

better not use relative import (see: Google Python Style Guide)